test/hotkey.test.ts

```diff
@@ -151,3 +151,108 @@ test('allows to override hotkeys', () => {
+  window.document.body.innerHTML =
+    '<ul>' +
+    '<li tabindex="0" data-keyux-ignore-hotkeys>' +
+    '<button aria-keyshortcuts="v">v1</button>' +
+    '</li>' +
+    '<li>' +
+    '<button aria-keyshortcuts="v">v2</button>' +
+    '</li>' +
+    '</ul>'
+
+  let clicked = ''
+  for (let button of window.document.querySelectorAll('button')) {
+    button.addEventListener('click', () => {
+      clicked += button.textContent
+    })
+  }
+
+  press(window, { key: 'v' })
+  equal(clicked, 'v2')
+
+  ;(window.document.querySelector('ul li') as HTMLLIElement).focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'v2v1')
+})
+
+test('should call element with "data-keyux-hotkeys" outside a container', () => {
+  let window = new JSDOM().window
+  startKeyUX(window, [hotkeyKeyUX()])
+  window.document.body.innerHTML =
+    '<ul>' +
+    '<li tabindex="0" data-keyux-hotkeys="click-on-third" data-keyux-ignore-hotkeys>' +
+    '<button aria-keyshortcuts="v">First button</button>' +
+    '</li>' +
+    '<li tabindex="0" data-keyux-ignore-hotkeys>' +
+    '<button aria-keyshortcuts="v">Second button </button>' +
+    '</li>' +
+    '</ul>' +
+    '<div data-keyux-ignore-hotkeys tabindex="0">' +
+    '<button aria-keyshortcuts="v" id="click-on-third">Third button </button>' +
+    '</div>'
+
+  let clicked = ''
+  for (let button of window.document.querySelectorAll('button')) {
+    button.addEventListener('click', () => {
+      clicked += button.textContent
+    })
+  }
+
+  press(window, { key: 'v' })
+  equal(clicked, '')
+
+  window.document.querySelectorAll('li')[0].focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Third button ')
+
+  window.document.querySelectorAll('li')[1].focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Third button Second button ')
+
+  window.document.querySelector('div')?.focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Third button Second button Third button ')
+})
+
+test('should support clicking on global element which occured before focused without ignore attr', () => {
+  let window = new JSDOM().window
+  startKeyUX(window, [hotkeyKeyUX()])
+  window.document.body.innerHTML =
+    '<ul>' +
+    '<li tabindex="0" data-keyux-ignore-hotkeys>' +
+    '<button aria-keyshortcuts="v">First button</button>' +
+    '</li>' +
+    '</ul>' +
+    '<button aria-keyshortcuts="v">Second button </button>' +
+    '<button>Focus element</button>'
+
+
+  let clicked = ''
+  for (let button of window.document.querySelectorAll('button')) {
+    button.addEventListener('click', () => {
+      clicked += button.textContent
+    })
+  }
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Second button ')
+
+  window.document.querySelectorAll('button')[1].focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Second button Second button ')
+
+  window.document.querySelectorAll('button')[2].focus()
+
+  press(window, { key: 'v' })
+  equal(clicked, 'Second button Second button Second button ')
+})
```

Comment:
Missed new line an the end.

If you install editorconfig extension to IDE, it will automatically resolve these issues.

Reply:
done